x/sisu/client/rest/gas_handler.go

```diff
@@ -0,0 +1,78 @@
+package rest
+
+import (
+	"encoding/json"
+	"github.com/sisu-network/sisu/x/sisu/helper"
+	"math/big"
+	"net/http"
+
+	"github.com/sisu-network/lib/log"
+)
+
+type gasCostRequest struct {
+	Chain   string `json:"chain,omitempty"`
+	TokenId string `json:"token_id,omitempty"`
+}
+
+type gasCostResponse struct {
+	Chain   string `json:"chain,omitempty"`
+	TokenId string `json:"token_id,omitempty"`
+	GasCost int64  `json:"gas_cost,omitempty"`
+}
+
+func (a *ExternalHandler) newGasCostHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		req := &gasCostRequest{}
+		if err := ReadRESTReq(r, req); err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		log.Info("Come here 1")
```

Comment:
Remove all the debug logs. Use .Debug function so that you can remove at the end.